.env.example

```
@@ -6,4 +6,6 @@
 # NEXT_PUBLIC_FIREBASE_APP_ID=your_appId
 # FIREBASE_SERVICE_ACCOUNT_KEY=your_serviceAccountKey
 NEXT_PUBLIC_SUPABASE_URL=http://localhost:1234
-NEXT_PUBLIC_SUPABASE_ANON_KEY=local-testing-key
\ No newline at end of file
+NEXT_PUBLIC_SUPABASE_ANON_KEY=local-testing-key
+SUPABASE_SERVICE_ROLE_KEY=local-testing-service-role-key
+GEMINI_API_KEY=your_api_key_here
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)